extensions/sql-migration/src/wizard/databaseBackupPage.ts

```diff
@@ -85,23 +85,35 @@ export class DatabaseBackupPage extends MigrationWizardPage {
 				checked: true
 			}).component();
 
-		networkShareButton.onDidClick((e) => this.toggleNetworkContainerFields(NetworkContainerType.NETWORK_SHARE, this._networkShare));
+		networkShareButton.onDidChangeCheckedState((e) => {
+			if (e) {
```

Comment:
This is only going to toggle the fields if e is true, don't you want to do it for false as well? (can we really send undefined here? That doesn't seem to make much sense for a boolean input)

Reply:
So, we are setting some variables in the model when it is true. When the other buttons are true, I am just overwriting the same variables. It is the something like 'last write wins (last write is by the button set to true)' kind of logic.

Reply:
Gotcha. That makes sense.